lib/controllers/commit-controller.js

```diff
@@ -31,6 +31,7 @@ export default class CommitController extends React.Component {
     stagedChangesExist: PropTypes.bool.isRequired,
     lastCommit: PropTypes.object.isRequired,
     currentBranch: PropTypes.object.isRequired,
+    mentionableUsers: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
it would increase code readability to use PropTypes.shape to specify what goes in this object